.buildkite/hooks/pre-command

```diff
@@ -3,5 +3,11 @@
 set -eu
 cd $BUILDKITE_BUILD_CHECKOUT_PATH
 
+if ! command -v asdf &> /dev/null
+then
+    echo "skipping asdf, TODO: ask Dave to add asdf to baremetal image"
```

Comment:
@davejrt can we pre-install the asdf command in the baremetal agent image? I suspect we'll need this in most CI pipelines that use it, so it would make sense to bake it in

Reply:
Going to leave this as-is until https://github.com/sourcegraph/sourcegraph/issues/12996 is fixed (which is higher priority).